nix/pkgs/kld.nix

```diff
@@ -50,6 +50,12 @@ craneLib.buildPackage {
     # having the tests seperate avoids having to run them on every package change.
     tests = craneLib.cargoTest {
       inherit src cargoToml cargoArtifacts buildInputs cargoExtraArgs outputHashes;
+      # FIXME: this copy shouldn't be necessary, but for some reason it tries to recompile openssl and fails
```

Comment:
@JosephGoulden Haven't found out why openssl-sys triggers a recompile (usually this can happen when some compiler/header files changes).
This is suboptimal because it now recompiles all crates for this derivation.